src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type TabsType = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}: {
+  tabs: TabsType[];
+  selectedTabId: string;
+  onTabSelected: React.Dispatch<React.SetStateAction<string>>;
+}) => {
+  return (
+    <>
+      {tabs.map((tab) => (
+        <li
+          key={tab.id}
+          data-cy="Tab"
+          className={classNames({ 'is-active': tab.id === selectedTabId })}
+        >
+          <a
+            href={`#${tab.id}`}
+            data-cy="TabLink"
+            id={tab.id}
+            onClick={(event) => onTabSelected(event.currentTarget.id)}
```

Comment:
First of all, move it to a separate function. And follow the requirements 
<img width="848" alt="image" src="https://user-images.githubusercontent.com/99403198/222889980-2d48874e-7a9b-42b4-950a-aa61a3b5046e.png">

Also, you don't really need an event here 🤔  You already have a `tab.id`